utils.js

```diff
@@ -4,8 +4,12 @@ const newsApi = axios.create({
     baseURL: 'https://nc-news-ku4r.onrender.com'
 })
 
-export const fetchArticles = () => {
-    return newsApi.get('api/articles')
+export const fetchArticles = (topic) => {
+    let url = 'api/articles'
+    if(topic.length > 0){
+        url += `?topic=${topic}`
+    }
+    return newsApi.get(url)
     .then((response) => {
         return response.data
     })
```

Comment:
This way is totally fine, but axios has its own way to deal with queries by passing an query object as the second argument, which will make it a bit easier then updating the url 
e.g. `newsApi.get("api/articles" , { params: { topic, sort_by, order ... } })`